src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -30,6 +34,9 @@ namespace SonarLint.VisualStudio.ConnectedMode.Migration
     [PartCreationPolicy(CreationPolicy.NonShared)]
     internal class ConnectedModeMigration : IConnectedModeMigration
     {
+        // Private "alias" to simplify method arguments
+        private sealed class ChangedFiles : List<FilePathAndContent<string>> { }
```

Comment:
Using the type directly got very ugly by the time it got wrapped in a task too:

`Task<List<FilePathAndContent<string>>>`